### PR TITLE
Update `swc_core`

### PR DESCRIPTION
**Pull request by kdy1**
Related:

 - https://github.com/swc-project/swc/pull/7069
 - https://github.com/swc-project/plugins/pull/158

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ serializable = ["serde"]
 [dependencies]
 markdown = "1.0.0-alpha.7"
 serde = { version = "1", optional = true }
-swc_core = { version = "0.59.0", features = [
+swc_core = { version = "0.69.2", features = [
```
Comment (wooorm) [inline Cargo.toml:21]:
```suggestion
swc_core = { version = "0.69.0", features = [
```

**Review by wooorm — APPROVED** [review]

**Comment by kdy1** [comment]
Can you publish a new version?

**Comment by wooorm** [comment]
yeah gimme a sec

**Comment by wooorm** [comment]
Released in [`0.1.8`](https://github.com/wooorm/mdxjs-rs/releases/tag/0.1.8)!

**Comment by kdy1** [comment]
Thank you!

**Pull request merged**